samples/word/99-preview-apis/manage-fields.yaml

```
@@ -0,0 +1,151 @@
+id: word-manage-fields
+name: Get fields
+description: This sample shows how to get existing fields.
+host: WORD
+api_set:
+    WordApi: '1.4'
+script:
+    content: |
+        $("#get-first").click(() => tryCatch(getFirstField));
+        $("#get-parent-body").click(() => tryCatch(getParentBodyOfFirstField));
+        $("#get-all").click(() => tryCatch(getAllFields));
+        $("#setup").click(() => tryCatch(setup));
+
+        async function getFirstField() {
+          await Word.run(async (context) => {
+            let field = context.document.body.fields.getFirstOrNullObject();
+            field.load(["code", "result"]);
+
+            await context.sync();
+
+            if (field.isNullObject) {
+              console.log("There are no fields in this document so there's no first field");
+            } else {
+              console.log("Code of first field: " + field.code);
+              console.log("Result of first field: " + field.result);
+            }
+          });
+        }
+
+        async function getParentBodyOfFirstField() {
+          await Word.run(async (context) => {
+            let field = context.document.body.fields.getFirstOrNullObject();
+            field.load("parentBody/text");
+
+            await context.sync();
+
+            if (field.isNullObject) {
+              console.log("There are no fields in this document so can't get parent body of non-existent first field");
+            } else {
+              let parentBody = field.parentBody;
+              console.log("Text of first field's parentBody: " + JSON.stringify(parentBody.text));
```

Consider writing this using the non-code phrasing.
```suggestion
              console.log("Text of first field's parent body: " + JSON.stringify(parentBody.text));
```